codes/ruby/chapter_array_and_linkedlist/array.rb

```diff
@@ -7,7 +7,7 @@
 ### 随机访问元素 ###
 def random_access(nums)
   # 在区间 [0, nums.length) 中随机抽取一个数字
-  random_index = Random.rand 0...(nums.length - 1)
+  random_index = Random.rand(0...nums.length)
```

Comment:
I fix this logic btw. 
- `0...nums.length` means from 0 to `nums.length - 1`
- `0..nums.length` means from 0 to `nums.length`
- `0...(nums.length - 1)` means from 0 to `nums.length - 2`